CourseManager/Views/SampleListView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SampleListView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-01.
+//
+
+import SwiftUI
+
+struct SampleListView: View {
+    
+    var sampleCode = SampleWork(name: "profile", description: "profile view")
+    var body: some View {
+        NavigationView {
+            ZStack {
+                LinearGradient(gradient: Gradient(colors: [AppColor.accent, .blue, .purple]), startPoint: .top, endPoint: .bottomLeading).ignoresSafeArea()
+                ScrollView {
+                    VStack {
+                        overlayContent
+                    }
+                }
+            }
+        }
+    }
+    var overlayContent: some View {
+        ForEach(sampleWork, id: \.id) { sample in
+            VStack(alignment: .leading, spacing: 12) {
+                Text(sample.name)
+                    .font(.largeTitle.weight(.bold))
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                Text(sample.description)
+                    .font(.footnote)
+
+                NavigationLink {
+                    SofaListView()
+                } label: {
+                    Text("View Example")
+                        .frame(maxWidth: .infinity)
+                }
+                .font(.headline)
+                .strokeStyle(cornerRadius: 20)
+                .buttonStyle(.angular)
+                .blendMode(.luminosity)
+                .tint(.accentColor)
+                .controlSize(.large)
+                .shadow(color: AppColor.shadow.opacity(0.2), radius: 5, x: 0, y: 1)
+            }
+            .padding(15)
+            .background(
+                Rectangle()
+                    .fill(.ultraThinMaterial)
+                    .mask(RoundedRectangle(cornerRadius: 20, style: .continuous))
+            )
+            .padding(.horizontal, 20)
+            .padding(.vertical, 8)
+        }
+    }
+}
+
+#if DEBUG
+#warning("To be moved to separate file after Model content created. This is just a prototype.")
+
+struct SampleWork: Identifiable {
```

Comment:
@sghourab create in your next PR , `SampleData.swift` and keep there all your sample data (like sofas, featuredCourses and etc)